tests/deps.ts

```diff
@@ -1,4 +1,4 @@
-export { Chats } from "https://ghc.deno.dev/dcdunkan/tests@v0.1.3/mod.ts";
+export { Chats } from "https://ghc.deno.dev/dcdunkan/tests@main/mod.ts";
```

Comment:
Is using a moving target smart?

Reply:
Not smart I am sure. But, https://t.me/grammyjs/61055.

I can change it to v0.1.4 if this is really not right.

Reply:
Done, changed back to v0.1.4